### PR TITLE
feature-show-one-task-per-page: added functionality to show one task per page to user

### LINKED ISSUE #113: Show one task per page
For now only one task is being displayed to user. When he submits it, he is not given further tasks. It can be resolved by using HTMX, ajax request.

### PR DESCRIPTION
**Pull request by najamF18**
Closes #113 
image

### CODE REVIEW

**Comment by turian** [comment]
@najamF18 where are your screenshots?

**Comment by najamF18** [comment]
> @najamF18 where are your screenshots?

just added them

File: polls/utils.py [hunk 1]
```diff
@@ -32,10 +32,46 @@ def check_user_work_permission(user):
...
+        return 0
+    percentage_gold = float(gold_task / total_task) * 100
+
+    return percentage_gold
+
+
+def get_user_roi(user):
+    """ """
+    pass
+
```
Comment (DeFidelity) [inline polls/utils.py:73]: From line 39 to 72, codes there are part of another pr that has been merged already, I suggest removal of it so as not to cause future conflict.

Comment (najamF18) [reply polls/utils.py:73]: Sure. I will look into it

File: polls/templates/polls/htmlform.html [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+  </audio>
+  <audio controls src="{{ transform_url }}" class="mt-3">
+    Your browser does not support the
+    <code>audio</code> element.
+  </audio>
+
+  <div class="mt-2">
+    <div class="form-check form-check-inline">
+      <input class="form-check-input" type="radio" id="type1" name="annotationOption" value="XXY" checked>
+      <label class="form-check-label" for="type1">XXY</label>
```
Comment (turian) [inline polls/templates/polls/htmlform.html:24]: XXY and XYX should not be displayed to users

File: polls/utils.py (hunk 1 above)
Comment (najamF18) [reply polls/utils.py]: Actually I merged changes in main into this branch. So, I think letting these be will not be an issue.

File: polls/templates/polls/htmlform.html (hunk 2 above)
Comment (turian) [reply polls/templates/polls/htmlform.html:24]: We should just have two radio boxes that are unlabeled.

Comment (najamF18) [reply polls/templates/polls/htmlform.html:24]: So, can we show text like "First and Second were same" or  "First and third sound is same"?

File: polls/utils.py (hunk 1 above)
Comment (DeFidelity) [reply polls/utils.py]: Okay, that's good then

**Review by DeFidelity — APPROVED** [review]

**Pull request merged**